### ISSUE TITLE
Checkboxes: updating model does not update view if type attribute is not the first attribute

### ISSUE THREAD

**Issue by gxrahul** [ractivejs/ractive#2336]
This is really weird wasted hours on it.
The following code does not work as expected. Model is getting updated, but checkboxes do not update. Only because "type" attribute is not the first one. If you put type="checkbox" before the "name" attribute it works fine.
http://jsfiddle.net/e8dnLd5r/
```
<div id="output"></div>
<script id="template" type="text/html">
    <input name="{{ shapes }}" type="checkbox" value="Circle" /> Circle<br/>
    <input name="{{ shapes }}" type="checkbox" value="Star" /> Star<br/>
    <input name="{{ shapes }}" type="checkbox" value="Square" /> Square<br/>
    <br/>
    Selected shapes: {{ shapes }}
</script>

<script type="text/javascript">
new Ractive({
    template: "#template",
    el: "#output",
    data: {
        shapes: ["Circle"]
    },
    onrender : function() {
    	var self = this;
    	setTimeout(function() {
      	self.set("shapes", ["Star"]);
      }, 2000);
    }
});
</script>
```

**Comment by MartinKolarik** [comment]
Fixed (#1968) and working in edge.

**Comment by nikhilvijayanv** [comment]
Its March 28. I can see issue closed on Jan 11. When will this issue be resolved in stable version ? I am using 0.7.3 .
